packages/client/test/miner/miner.spec.ts

```diff
@@ -325,11 +329,25 @@ tape('[Miner]', async (t) => {
     txPool.start()
     miner.start()
 
-    await setBalance(vm.stateManager, A.address, new BN('200000000000001'))
+    let pkey = keccak256(Buffer.from(''))
 
     // add many txs to slow assembling
     for (let i = 0; i < 1000; i++) {
-      txPool.add(createTx())
+      // In order not to pollute TxPool with too many txs from the same address
+      // (or txs which are already known), keep generating a new address for each tx
+      const address = new Address(privateToAddress(pkey))
```

Comment:
btw we have this method
```suggestion
      const address = Address.fromPrivateKey(pkey)
```